net/hev-socks5-tproxy/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hev-socks5-tproxy
+PKG_VERSION:=2.5.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/heiher/hev-socks5-tproxy/releases/download/$(PKG_VERSION)
+PKG_HASH:=b5f29fef18ffe335fce4d6f96ce83c8bee04ce29fda420ba7248252d3b210578
+
+PKG_MAINTAINER:=Ray Wang <r@hev.cc>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=License
+
+PKG_BUILD_FLAGS:=no-mips16
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/hev-socks5-tproxy
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=A simple, lightweight socks5 transparent proxy for Linux
+  URL:=https://github.com/heiher/hev-socks5-tproxy
+endef
+
```

Comment:
Can we have description for this package, please?

Reply:
I have already added it. Thanks.